app/models/user.rb

```diff
@@ -33,17 +36,15 @@ def queued_video?(video)
   end
 
   private
+    def new_queue_item_review(queue_item, rating)
+      queue_item.video.reviews.create!(skip_validation: true, rating: rating, author: queue_item.user)
+    end
 
-  def new_queue_item_review(queue_item, rating)
-    queue_item.video.reviews.create!(skip_validation: true, rating: rating, author: queue_item.user)
-  end
-
-  def update_queue_item_rating(queue_item, rating)
-    queue_item.video.reviews.where(author: queue_item.user).first.update!(skip_validation: true, rating: rating)
-  end
-
-  def input_rating_empty?(rating)
-    rating == ""
-  end
+    def update_queue_item_rating(queue_item, rating)
+      queue_item.video.reviews.where(author: queue_item.user).first.update!(skip_validation: true, rating: rating)
```

Comment:
That's a good point, I will separate finding the review and if it exists then updating it.  Thanks!